### ISSUE TITLE
Bookworm

### ISSUE THREAD

**Issue by pcfreaxx** [mordesku/PS4-Debian-Drivers#1]
I tried today to Install in bookworm. Are there any Tips? It failed due missing dependencys

**Comment by mordesku** [comment]
Nope, these were made using Debian 11 sources. I'll build a new version soon.

**Comment by mordesku** [comment]
@pcfreaxx I can build the packages today, but I can't test them anytime soon. Would you like to give it a try first?

**Comment by pcfreaxx** [comment]
Yeah. Would be a pleasure.

**Comment by mordesku** [comment]
The packages are ready.

**Comment by pcfreaxx** [comment]
Looks good so far. Internal Install: aeiola, 5.15.15 codedrench, initram by nazky.
Little blackscreen issue at boot. Ethernet thinks it is working but it dont😁

This night i try the whole gaming stuff.
Many thx for this.💪
I wish i could build drivers like you.

**Comment by mordesku** [comment]
Building packages is easy. You need to follow a couple of steps. Patching might be tricky. Anyway, how did the game test go?

**Comment by pcfreaxx** [comment]
Tried a few. No Proton worked tested from 7.50 to 8.4.
Do you need any Output? I tried 1:1 to repeat noob404's way to start with ge-proton7.50 to start Prince of Persia 2008. Screenshot will follow

**Comment by mordesku** [comment]
So first, please check if mesa is working by typing `vulkaninfo` into a terminal. You can redirect the output to the log file and attach it here. To do so, type `vulkaninfo &>vulkaninfo.txt`. 
You can also run glgears from mesa-utils package.

**Comment by pcfreaxx** [comment]
[vulkaninfo.txt](https://github.com/mordesku/PS4-Debian-Drivers/files/11879533/vulkaninfo.txt)
then i build custom mesa 23.1.3 and made another vulkaninfo

[vulkaninf_aftercustom_mesa.txt](https://github.com/mordesku/PS4-Debian-Drivers/files/11879545/vulkaninf_aftercustom_mesa.txt)

**Comment by mordesku** [comment]
In both cases, the driver is working, it properly detects the PS4 GPU. Your problems seem to be related to other issues. Have you installed i386 mesa and libdrm? As far as I remember It is crucial for running 32bit Windows apps on Wine.

**Comment by mordesku** [comment]
https://wiki.debian.org/Wine this might be useful.
